scripts/baseline_capture.sh

```
@@ -0,0 +1,81 @@
+    echo "## Playbooks:"
+    find . -name "*.yml" -o -name "*.yaml" | grep -i playbook | sort || echo "No playbooks found"
+    echo ""
+    
+    # Find all roles
+    echo "## Roles:"
+    find . -type d -name "roles" -exec find {} -mindepth 1 -maxdepth 1 -type d \; | sort || echo "No roles found"
+    echo ""
+    
+    # Find all collections references
+    echo "## Collections References:"
+    find . -name "*.yml" -o -name "*.yaml" | xargs grep -l "collections:" 2>/dev/null | sort || echo "No collection references found"
```

Shellcheck (0.11.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)